core/templates/dev/head/footer.html

```diff
@@ -36,7 +36,7 @@ <h4 translate="I18N_FOOTER_CONTRIBUTE_ALL_CAPS"></h4>
         <ul ng-controller="I18nFooter">
           <select ng-model="currentLanguageCode"
                   ng-change="changeLanguage()"
-                  ng-options="langCode as langName for (langCode, langName) in supportedSiteLanguages"
+                  ng-options="language['id'] as language['text'] for language in supportedSiteLanguages"
```

Comment:
Prefer using `language.id` instead of `language['id']`. Similar for `language.text`.